### DISCUSSION TITLE
Messages with Mr. Browner

### DISCUSSION THREAD

**Discussion by MrBrowner** [girishg4t/bhavCopy-downloader#38] [General]
Hello, Girish.

Long time, no see.
How are you?

**Comment by MrBrowner** [comment]
I want to start making contributions to open projects.

And, looks like you are making updates to this project.
So, won't mind if we connect & discuss about the changes.
Or we can discuss here.

Regards,
MrBrowner

**Reply by girishg4t** [reply]
I am good,
you can start integrating any of this api's 


MARKET_STATUS_URL = 'https://www1.nseindia.com//emerge/homepage/smeNormalMktStatus.json';
INDICES_WATCH_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/stock_watch/liveIndexWatchData.json';
SECTORS_LIST = 'https://www1.nseindia.com/homepage/peDetails.json';
QUOTE_INFO_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/get_quote/ajaxGetQuoteJSON.jsp?series=EQ&symbol=';
GET_QUOTE_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/get_quote/GetQuote.jsp?symbol=';
GAINERS_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_analysis/gainers/niftyGainers1.json';
LOSERS_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_analysis/losers/niftyLosers1.json';
ADVANCES_DECLINES_URL = 'https://www1.nseindia.com/common/json/indicesAdvanceDeclines.json';
INDEX_STOCKS_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/stock_watch/';
INTRADAY_URL = 'https://www1.nseindia.com/charts/webtame/tame_intraday_getQuote_closing_redgreen.jsp?Segment=CM&Series=EQ&CDExpiryMonth=&FOExpiryMonth=&IRFExpiryMonth=&CDDate1=&CDDate2=&Template=tame_intraday_getQuote_closing_redgreen.jsp&CDSymbol=';
SEARCH_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/get_quote/ajaxCompanySearch.jsp?search=';
STOCK_OPTIONS_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_watch/get_quote/ajaxFOGetQuoteJSON.jsp';
YEAR_HIGH_URL = 'https://www1.nseindia.com/products/dynaContent/equities/equities/json/online52NewHigh.json';
YEAR_LOW_URL = 'https://www1.nseindia.com/products/dynaContent/equities/equities/json/online52NewLow.json';
TOP_VALUE_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_analysis/most_active/allTopValue1.json';
TOP_VOLUME_URL = 'https://www1.nseindia.com/live_market/dynaContent/live_analysis/most_active/allTopVolume1.json';
NEW_CHART_DATA_URL = 'https://www1.nseindia.com/ChartApp/install/charts/data/GetHistoricalNew.jsp';

**Reply by MrBrowner** [reply]
I'm having trouble understanding the code. 😓

Can i get the sample code/file where we integrated an API like this.

**Comment by MrBrowner** [comment]
Also, I noticed that in 17JUL2024 file.

The data-set values are of diff date's (i.e. 16th Jul 2024).
In excel, the quotes @ end of each cell block!

![Screenshot 2024-07-28 124341](https://github.com/user-attachments/assets/c1ebb223-3c61-4e65-b33c-fe63ebf063b5)

**Reply by girishg4t** [reply]
will have a look
